README.md

```diff
@@ -120,6 +120,7 @@ These consultants use the Serverless Framework and can help you build your serve
 * [SC5 Online](https://sc5.io)
 * [Carrot Creative](https://carrot.is)
 * [microapps](http://microapps.com)
+* [Obie Fernandez](http://obiefernandez.com) author of the definitive serverless [book](https://leanpub.com/serverless)
```

Comment:
"author of the definitive serverless" would sound like we specifically call it the definitive one, which we can't do. Something like "Author of Serverless - Patterns of Modern Application Design Using Microservices" would be fine.